operators/src/adapters/raster_subquery/raster_subquery_reprojection.rs

```diff
@@ -144,6 +116,98 @@ where
     }
 }
 
+fn projected_coordinate_grid_parallel(
+    pool: &ThreadPool,
+    tile_info: TileInformation,
+    out_srs: SpatialReference,
+    in_srs: SpatialReference,
+    valid_out_area: &SpatialPartition2D,
+) -> Result<Grid2D<Option<Coordinate2D>>> {
+    const OVERPROVISIONING: usize = 1; // split the work into multiple chunks per thread.
+    const MAX_PARALELLISM: usize = 8; // max task / work split
+
+    let start = std::time::Instant::now();
+
+    let res = pool.install(|| {
```

Comment:
Do you have this wrapped into a `spawn_blocking` at some point where this is called?

Reply:
no and we know that this is an issue. There is a case missing in the subquery adapter. Here we need to await the accu creation:  https://github.com/geo-engine/geoengine/blob/624e2a7461f243271cae391b3c7b19c6929210f7/operators/src/adapters/raster_subquery/raster_subquery_adapter.rs#L312


however. this is not in the scope of this PR

Reply:
i changed this in 22e6994